tools/parity_scale_cli/src/main.rs

```diff
@@ -328,9 +326,7 @@ mod tests {
     #[test]
     fn decode_domain_sample() {
         let mut metadata = Metadata::default();
-        metadata
-            .insert("Is_Jabberwocky_alive".parse().expect("Valid"), true)
-            .expect("Valid");
+        metadata.insert("Is_Jabberwocky_alive".parse().expect("Valid"), true);
```

Comment:
Previously there was method like `insert_with_limits` which returns error if value exceeds limits. Now `insert` returns old value so `expect` not needed